docs/build/tooling/foundry/overview.md

```diff
@@ -0,0 +1,53 @@
+---
+head:
+  - - meta
+    - name: "twitter:title"
+      content: Foundry | zkSync Docs
+---
+
+# Foundry with zkSync
+
+## Overview
+
+`foundry-zksync` is a specialized fork of [Foundry](https://github.com/foundry-rs/foundry), tailored for zkSync. It extends Foundry's capabilities for Ethereum app development to support zkSync, allowing for the compilation, deployment, testing, and interaction with smart contracts on zkSync. `foundry-zksync` introduces `zkforge` and `zkcast` extensions of Foundry's existing `forge` and `cast` tools but tailored for zkSync usage.
```

Comment:
Perhaps this is something more for zkfoundry README, but we should document which upstream commit/version we forked

Reply:
Let me make this change.